gbfile.cc

```diff
@@ -538,8 +538,7 @@ gbfopen(const QString filename, const char* mode, const char* module)
     file->fileungetc = memapi_ungetc;
     file->filewrite = memapi_write;
   } else {
-    /* Be careful to convert back to local8Bit for these c based APIS */
-    file->name = xstrdup(qPrintable(filename));
```

Comment:
If you merge up travis-ci should show a problem when it runs test_encoding due to not converting back to local8bit.